testpilot/experiments/migrations/0018_auto_20160705_1355.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('experiments', '0017_discourse_url'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='featurecondition',
+            name='comment',
+            field=models.TextField(blank=True),
+        ),
+        migrations.AlterField(
+            model_name='featurecondition',
+            name='operator',
+            field=models.CharField(choices=[('', 'None')], max_length=256),
+        ),
+    ]
```

Comment:
This looks related to the discourse_url change that landed recently? I'm guessing it can be deleted too